src/i18n/fi/translation.json

```diff
@@ -234,7 +234,7 @@
       "heading-1": "Mitä minä haluan",
       "map-competence": "Kartoittaa omaa osaamistani",
       "trainings": "Koulutuksia  osaamiseni  kasvattamiseksi",
-      "job-opportunities": "Koulutuksia osaamiseni kasvattamiseksi",
+      "job-opportunities": "Tutkia  työmahdollisuuksia osaamisellani",
```

Comment:
Two spaces

Reply:
Good find. Should not have copied the value from the Figma :D